### PR TITLE
fix the go version number error.

### PR DESCRIPTION
**Pull request by Jingshui1037**
Currently, go.mod need go 1.20(https://github.com/google/go-tpm-tools/blob/master/go.mod#L4),
so we need to update the go version number to go 1.20.

Fixes: #327 

Signed-off-by: YiLin.Li <YiLin.Li@linux.alibaba.com>
Signed-off-by: Jingshui1037 <jingshui1037@163.com>

### CODE REVIEW

**Review by jkl73 — APPROVED** [review]

**Review by jkl73 — COMMENTED** [review]
You can actually delete this comment

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/google/go-tpm-tools
 
-// Move to new-style build tags when minimum supported Go version is 1.17
+// Move to new-style build tags when minimum supported Go version is 1.20
```
Comment (jkl73) [inline go.mod:3]: You can actually delete this comment

**Comment by Jingshui1037** [comment]
Hi @jkl73 ,

Thank you very much for your review, I have modified it. Could you review it again?

**Comment by jkl73** [comment]
Seems like one test failed because of `"/lib64/libc.so.6: version GLIBC_2.34 not found"`


Could you update this line
https://github.com/google/go-tpm-tools/pull/326/files#diff-ac8d8fc3e2ed1623cdc3f10cde2f98188ab33df1a49122b92991a3c83a95a34cL17
to `CGO_ENABLED=0 go build -o ../image/launcher`

Hopefully this can fix the issue.

**Comment by Jingshui1037** [comment]
Hi @jkl73 ,

Thank you very much for your review, I have modified it. But I can't re-run the CI, Could you help me?

**Review by jkl73 — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**